submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/MultipartFileSubmissionApiTest.kt

```diff
@@ -176,7 +139,7 @@ internal class MultipartFileSubmissionApiTest(
                         "value" to "FileList.json"
                     })
                 }
-            }
+            }.toString()
 
             val fileList = tempFolder.createFile("FileList.json").apply {
                 writeBytes(jsonArray({
```

Comment:
I create an overload that accept file and content so I think we can use everywhere it looks way more clean IMO.

```
 val fileList = tempFolder.createFile("FileList.tsv").apply {
                writeBytes(tsv {
                    line("Files", "GEN")
                    line("SomeFile.txt", "ABC")
                }.toString().toByteArray())
            }
```

```
val fileList = tempFolder.createFile(
    "FileList.tsv", 
     tsv {
            line("Files", "GEN")
            line("SomeFile.txt", "ABC")
        }.toString())
```

Reply:
Done